bluesky/ophyd-async
# Fastcs eiger support

See https://github.com/DiamondLightSource/dodal/issues/700. This implements enough of the Eiger/Odin for us to prove that we can actually take some data with it. To completely cover all the capability of the `ophyd` device we need to do a lot, at least:

* https://github.com/bluesky/ophyd-async/issues/529
* https://github.com/bluesky/ophyd-async/issues/530
* https://github.com/bluesky/ophyd-async/issues/531
* https://github.com/bluesky/ophyd-async/issues/532

# To Test
* Confirm unit tests pass
* Run the system tests as described in the system test directory, confirm they pass

# Notes
* I'm particularly interested in someone who knows the intent of the detector architecture to review this to confirm it follows the pattern correctly.
